test/ably/restauth_test.py

```diff
@@ -510,6 +510,7 @@ def test_client_id_null_until_auth(self):
 class TestRenewToken(BaseTestCase):
 
     def setUp(self):
+        host = test_vars['host']
```

Comment:
In other libs, we normally allow the env for testing to be set with an env var such as `ABLY_ENV=staging` so that you can run your tests easily against non-sandbox default envs